docs/src/pages/getting-started/faq/faq.md

```diff
@@ -443,3 +443,8 @@ return (
   />
 );
 ```
+
+## I added a `ThemeProvider`. Why does the styling not change accordingly?
+
+Styling the components per [`ThemeProvider`](https://mui.com/customization/theming/#theme-provider) requires [`@emotion/styled`](https://www.npmjs.com/package/@emotion/styled).
```

Comment:
Rendering any component requires `@emotion/styled` and `@emotion/react`, if you use the default stylied engine no? How is it related to the `ThemeProvider` usage?

Reply:
No, `@emotion/react` is sufficient. Components render and you can test and build packages, import them in some projects and if `@emotion/styled` is present there also theming works fine in the end result.

But when I added themes to the Storybook of a component library, at first glance I couldn't figure out why the `ThemeProvider` wasn't working there. As using `@emotion` powered packages in Storybook already can require some workarounds because of conflicting versions, I think it was worth mentioning that this is another thing one needs to check when things aren't working as expected

Reply:
I'm facing the same problem that custom them is not working on Storybook. It's still a valid problem. I've spent hours trying to find the solution. I followed the conversation on this thread and the custom style is working properly in the application we are building but it's now working on Storybook. Any hints?

Reply:
Did you check this guide - https://mui.com/guides/migration-v4/#storybook-emotion-with-v5 I think the issue is related to it.

Reply:
@rapitkan 
Storybook requires an older version of `@emotion/styled` (and `@emotion/react`). This results in a broken build with multiple emotion versions.
But it "just works" with the recent version no matter what its `package.json` says.

You can override the webpack config used by Storybook in `.storybook/main.js`:
```js
    webpackFinal: async(config) => ({
        ...config,
        resolve: {
            ...config.resolve,
            alias: {
                '@emotion/styled': require.resolve('@emotion/styled'),
            },
        },
    }),
```